packages/framework/src/Controller/Admin/AdministratorController.php

```diff
@@ -125,6 +134,10 @@ public function editAction(Request $request, $id)
             try {
                 $this->administratorFacade->edit($id, $administratorData);
 
+                if ($administrator->getId() === (int)$id) {
```

Comment:
couldn't be typehint in method argument?
```diff
-  public function editAction(Request $request, $id)
+  public function editAction(Request $request, int $id)
```

Reply:
fixed